frontend/src/components/Modal.js

```diff
@@ -1,17 +1,21 @@
 import React from "react";
+//import { RiCloseLine } from "react-icons/ri";
 
-const Modal = () => {
+const Modal = ({ setIsOpen, responseMessage }) => {
```

Comment:
This file needs to be renamed to something more descriptive, we will have many modals throughout the project.